### DISCUSSION TITLE
Can this text box be turned grey?

### DISCUSSION THREAD

**Discussion by Andre601** [mastodon/mastodon#20187] [Q&A] [answered]
Recently did I switch my account to another site which uses v4 of Mastodon to actually use some of the QOL changes.

Looking at it, I noticed that the text box is now a bright white, even when using a dark theme here.
![grafik](https://user-images.githubusercontent.com/11576465/200704964-3df72b3e-ab1d-4e9f-8c05-ca8563c7447f.png)

Idk if the previous site I used had some custom CSS that changed the text box to a dark grey, or if this in fact a change from v3 to v4, but for me is this an eyesore to put it bluntly.
Not only is it sticking out from the rest of the theme, making it look out of place, but it also hurts in my eyes looking too long at it, which I may have to do if I write a post.

So I'm now asking if there is some way to change this style on the user end, or if the site admins need to do something here.

**Comment by MrGeneration** [comment]
Well technically your instance can actually add custom CSS within the site administration.
However, this CSS counts for all 3 theme variants. So having that seperated cleanly is not that trivial but doable.

On a 3.5.3 I "quickly" hacked around the following. It explictely is not finished and is missing fine tuning and some parts of the affected UI. Here's the result

![image](https://user-images.githubusercontent.com/6549061/200717712-7ce54af3-8ffa-4fa3-aafc-8ffbbc740af1.png)
![image](https://user-images.githubusercontent.com/6549061/200717759-07081c02-9cf3-473a-b7ce-3c04b7a7615c.png)
![image](https://user-images.githubusercontent.com/6549061/200717796-9ee8e615-1490-411c-9bb1-3f591d6c78a5.png)
![image](https://user-images.githubusercontent.com/6549061/200717823-ef43c6f3-12a6-4123-8555-96796ad19a0a.png)

You see, not ready exactly.

However, if one would want to use it, this would be the CSS that would go to custom CSS:
```
.theme-default .compose-panel .autosuggest-textarea__textarea,
.theme-default .compose-form .compose-form__modifiers,
.theme-default .autosuggest-textarea__suggestions,
.theme-default .compose-form__buttons-wrapper,
.theme-default .compose-form input {
    background-color: #282c37 !important;
    color: #fefefe !important;
}

.theme-default .compose-form  .compose-form__modifiers span,
.theme-default .compose-form  .compose-form__sensitive-button span {
    color: #eeee;
}

.theme-default .emoji-mart-scroll {
    background-color: #404040;
}

.theme-default .emoji-mart-category-label span {
    background-color: #545454 !important;
    color: #eee;
}

.theme-default .emoji-mart-category .emoji-mart-emoji:hover:before {
    background-color: #727272;
}

.theme-default .emoji-picker-dropdown__menu {
    background-color: #2D2D2D;
}

.theme-default .emoji-mart-search {
    background-color: transparent;
}

.theme-default .emoji-mart-bar {
    background-color: #404040 !important;
}

.theme-default .emoji-mart-anchor-icon {
    color: #AFAFAF;
}


```

**Reply by Andre601** [reply]
Alright, thanks for the info and possible fix.
Due to what trwnh commented will I wait for the full v4 update to be released to see if things change further.

**Comment by trwnh** [comment, marked as answer]
#19205 was submitted but it seems that a larger theme re-structuring may be undertaken after 4.0
